cli/src/main/java/org/jboss/gm/cli/Main.java

```diff
@@ -298,5 +298,17 @@ public int handleExecutionException(Exception ex, CommandLine commandLine, Comma
             this.exception = ex;
             return 1;
         }
+
+        public Exception getException() {
+            if (exception != null) {
+                if ("Caught exception running build".equals(exception.getMessage())) {
+                    exception = (Exception) exception.getCause();
+                }
+                if ("org.gradle.internal.exceptions.LocationAwareException".equals(exception.getClass().getName())) {
```

Comment:
I think it's more correct and more performant to use `if (exception instanceof org.gradle.internal.exceptions.LocationAwareException)` here, although I find this pattern in two other places in the code as well.

Reply:
Except that class isn't in the module classpath

Reply:
Also it's on a fail path in the code - the app is about to exit so it really shouldn't matter at that point.